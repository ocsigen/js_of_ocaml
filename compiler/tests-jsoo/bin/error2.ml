let () =
  match Array.to_list Sys.argv with
  | _ :: "unregister" :: _ ->
      let null = Array.unsafe_get [| 1 |] 1 in
      Callback.register "Printexc.handle_uncaught_exception" null
  | _ -> ()

[@@@ocaml.warning "-8"]

let _ =
  match 3 with
  | 2 -> ()
