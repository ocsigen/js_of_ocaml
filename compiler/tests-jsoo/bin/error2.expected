Fatal error: exception File "compiler/tests-jsoo/bin/error2.ml", line 11, characters 2-7: Pattern matching failed
